tests/regr_test.py

```diff
@@ -315,15 +334,20 @@ def main() -> ReturnCode:
         platforms_to_test, versions_to_test = SUPPORTED_PLATFORMS, SUPPORTED_VERSIONS
     else:
         platforms_to_test = args.platforms_to_test or [sys.platform]
-        versions_to_test = args.versions_to_test or [f"3.{sys.version_info[1]}"]
+        versions_to_test = args.versions_to_test or [PYTHON_VERSION]
 
     code = 0
+
     results: list[Result] | None = None
 
     with ExitStack() as stack:
         results = concurrently_run_testcases(stack, testcase_directories, verbosity, platforms_to_test, versions_to_test)
 
     assert results is not None
+    if not results:
+        print(colored("All tests were skipped!", "yellow"))
+        return 0
```

Comment:
Hmm. I think we should return a nonzero exit code if _all_ tests were skipped. (This also applies to `mypy_test.py` and `stubtest_third_party.py`.) It's probably not what the user was expecting when they ran the script. If you're just running it from the command line, you'll notice that they've all been skipped due to the verbiage printed to the terminal, so it's okay. But if this is being run in CI, we could have all tests being skipped (due to something going wrong somewhere, idk), and the job would be marked as a success.